app/src/main/java/com/kylecombes/micrepeater/MainActivity2.java

```diff
@@ -111,10 +111,20 @@ public void stateChanged(boolean deviceConnected, boolean scoAudioConnected,
     public void startAmplification() {
         Log.d(TAG, "Starting amplification...");
         audioRelayServiceIntent = new Intent(this, AudioRelayService.class);
-        Integer streamType = Objects.requireNonNull(mViewModel.getStreamType().getValue());
-        audioRelayServiceIntent.putExtra(AudioRelayService.STREAM_KEY, streamType);
-        // Set default volume control
-        setVolumeControlStream(streamType);
+        String streamType = getSharedPreferences("PREFERENCE", MODE_PRIVATE)
+                .getString("audioOutput", "voice");
+        if(streamType.equals("voice")){
```

Comment:
there should be a space between `if` and the parenthesis and between the parenthesis and `{`.
```
if (condition) {
```

Same for below. If you don't want to fix these manually. You can use `Ctrl + Alt + L` to have the IDE fix it for you automatically.